Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs

```diff
@@ -0,0 +1,99 @@
+﻿using gax = Google.Api.Gax;
```

Comment:
No copyright - I may just not have noticed this from previous files. Is that still to-do in general?